### PR TITLE
Message Pagination Front end

### PR DESCRIPTION
**Pull request by Mari-Hayashi**
Added "Load more" button at the top of the chat area so that users can read old messages.
MessagePagination

### CODE REVIEW

File: src/client/src/Domains/Messages/GetMessages/index.tsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import Grid from '@material-ui/core/Grid';
+import Button from '@material-ui/core/Button';
```
Comment (johan1505) [inline src/client/src/Domains/Messages/GetMessages/index.tsx]: Use the component in Components/Button.

File: src/client/src/Domains/Messages/GetMessages/index.tsx [hunk 2]
```diff
@@ -63,16 +78,29 @@ function Messages({ receiver }: MessagesProps) {
...
+                            </Grid>
                     ) : (
-                        messages.map((message, index) => (
-                            <Grid item key={index} xs={12}>
-                                <MessageDialog message={message} />
+                            <Grid item>
+                                {messageCount > page * 20 &&
+                                    <Button
+                                        onClick={loadMoreMessage}
+                                        color='primary'
```
Comment (johan1505) [inline src/client/src/Domains/Messages/GetMessages/index.tsx]: Using the Button from the Components folder, you don't need to pass the color = 'primary' or variant='contained' as they are the default properties

**Review by johan1505 — CHANGES_REQUESTED** [review]
left comments

File: src/client/src/Domains/Messages/GetMessages/index.tsx (hunk 1 above)
Comment (Mari-Hayashi) [reply src/client/src/Domains/Messages/GetMessages/index.tsx]: Done.

File: src/client/src/Domains/Messages/GetMessages/index.tsx (hunk 2 above)
Comment (Mari-Hayashi) [reply src/client/src/Domains/Messages/GetMessages/index.tsx]: Done.

**Pull request merged**